pkg/cluster/https.go

```diff
@@ -61,6 +62,8 @@ func (c *Cluster) initClusterAndHTTPS(ctx context.Context) error {
 
 	server := http.Server{
 		Handler: handler,
+		ErrorLog: log.New(logrus.StandardLogger().Writer(), "Cluster-Http-Server", log.LstdFlags),
```

Comment:
Based on the log output shared here: https://github.com/rancher/k3s/pull/2242#discussion_r488190392
It looks like we may need a space at the end of the prefix afterall because it is getting smushed into the timestamp.